src/Components/form.jsx

```diff
@@ -0,0 +1,11 @@
+function Form() {
+  return (
+    <form>
+      <input type="text" placeholder="Book Title" required />
+      <input type="text" placeholder="Book Title" required />
```

Comment:
- [ ] The second input should have a placeholder for author, not title. Kindly fix it.